dev_docs/getting_started/hello_world_plugin.mdx

```diff
@@ -40,6 +40,10 @@ and add the following:
   "id": "helloWorld",
   "version": "1.0.0",
   "kibanaVersion": "kibana",
+    "owner": {
+    "name": "My Team",
```

Comment:
```suggestion
    "name": "Kibana Core",
```

Reply:
Perfect thank you. I made those changes!